From eb1b744102ae0a88344f1b56165b1d62cb54b3c9 Mon Sep 17 00:00:00 2001
From: Ruslan Baratov <x@ruslo.dev>
Date: Sat, 18 May 2024 23:34:19 +0800
Subject: [PATCH 104/228] [Docs] Default value for verification flags is
 'SSL_VERIFY_NONE'

Reviewed-by: Paul Dale <ppzgs1@gmail.com>
Reviewed-by: Tomas Mraz <tomas@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/24435)

(cherry picked from commit a73e07dbb7df4795c4ec537f19516b541fb8dd3c)
---
 doc/man3/SSL_CTX_set_verify.pod | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/man3/SSL_CTX_set_verify.pod b/doc/man3/SSL_CTX_set_verify.pod
index 9d4abac30e..fcbf276798 100644
--- a/doc/man3/SSL_CTX_set_verify.pod
+++ b/doc/man3/SSL_CTX_set_verify.pod
@@ -144,6 +144,9 @@ B<Client mode:> ignored (see BUGS)
 
 If the B<mode> is SSL_VERIFY_NONE none of the other flags may be set.
 
+If verification flags are not modified explicitly by C<SSL_CTX_set_verify()>
+or C<SSL_set_verify()>, the default value will be SSL_VERIFY_NONE.
+
 The actual verification procedure is performed either using the built-in
 verification procedure or using another application provided verification
 function set with
-- 
2.39.5 (Apple Git-154)

